### DISCUSSION TITLE
Self-host. In the dashboard Functions, deployments, backups, logs, settings are disabled.

### DISCUSSION THREAD

**Discussion by nokia6290** [nhost/nhost#2498] [Q&A]
Hello,

Self-host. In the dashboard Functions, deployments, backups, logs, settings are disabled.

How to enable them..?
![Screenshot 2024-01-28 at 9 15 00 pm](https://github.com/nhost/nhost/assets/13733620/89b8c766-4714-4d17-ac68-771bc089f4a6)

**Comment by nokia6290** [comment]
![Screenshot 2024-01-28 at 9 22 08 pm](https://github.com/nhost/nhost/assets/13733620/a3d0e4b7-93e2-4b85-85bf-2bbbcf371359)

database - broken ? what else need to be modified in order to create a new table..?

I'm confused. Is the self-host docker image broken or requires some hidden configurations ?

**Comment by dbarrosop** [comment]
Those are functions of our cloud and won't work anywhere else so they can't be enabled outside of our cloud service.

We don't support self-hosted solutions (without a support agreement) as there are a million ways things can be configured and they all can break in different ways. I'd suggest looking at the developer tools and the available environment variables to configure the dashboard, that might give you a clue on what's wrong.

**Reply by nokia6290** [reply]
I was not aware that there are limits on self-host.

Could you please explain what is available to self-host then?

Also can you explain why the database is not working?

**Reply by dbarrosop** [reply]
there are no limits on self-host but the sections that are disabled are functionality provided by our cloud (i.e. deployments are going to be specific to your infrastructure of choice, same for metrics and logs).

No, I don't know why your database is not working as I don't have access to your self-hosted project. You should ask your database administrator, they may be able to help.

**Reply by nokia6290** [reply]
Has anyone done any progress on this? Or nhost self-host is similar to supabase - 'kind of fake self-host' ?

**Reply by GFoley83** [reply]
Seems a lot of people are having the same issues while self-hosting. The only official support is the docker-compose which seems to be a little flaky/poorly tested.

**Reply by kevin4dhd** [reply]
> Has anyone done any progress on this? Or nhost self-host is similar to supabase - 'kind of fake self-host' ?

Have you had any luck? It's been 6 months since your comment, you kept trying and you're not using nhost anymore? Docker compose auto self hasn't been updated for 1 year
